data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,457 @@
+	destination "Darkcloak"
+
+
+mission "Unfettered Silicon Asteroid Mining"
+	name `Gather silicon from asteroids`
+	description `Mine and then collect 50 tons of Silicon from the asteroid field in Wah Yoot and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Darkcloak"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 250000
+		dialog `The Unfettered unloading the materials you mined seem enthused.	"This will allow us to repair our ships!" they tell you and hand over your payment of <payment> credits.`
+		outfit "Silicon" -50
+
+mission "Unfettered Silver Asteroid Mining"
+	name `Gather silver from asteroids`
+	description `Mine and then collect 15 tons of Silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Firelode"
+	destination "Darkcloak"
+	waypoint "Ehma Ti"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 400000
+		dialog `The Unfettered unloading the materials you mined seem satisfied.	"This will allow us to repair our ships!" they tell you and hand over your payment of <payment> credits.`
+		outfit "Silver" -15
+
+
+
+mission "Unfettered Medicals"
+	name `Express medical supplies to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	job
+	repeat
+	deadline 10 1
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		random < 80
+	cargo "medicals" 15 2 .05
+	stopover
+		distance 5 7
+		government "Hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong,"	they ask while loading the supplies on your ship.`
+	on complete
+		payment 10000 4000
+		dialog `The local health specialists seem relieved that you have managed to bring the needed medical supplies back in time.`
+
+
+
+mission "Unfettered Challenge[0]"
+	name `Defeat the young captain`
+	description `A young Unfettered captain has provoked you in a duel, disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 60
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Pulse)"
+		conversation
+			action
+				payment 200000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 15
+				fail "Unfettered Challenge[0]"
+			`You receive a message from the <npc> : "Raaaah... Well you have beaten me alright, so I'll pay up... Here are the <payment> credits, captain."`
```

Comment:
Would it make sense for the Unfettered here to react this way? When you disable Unfettered ships in normal battle, their hails are extremely hostile to you. I wouldn't expect an Unfettered to be so calm after defeat, considering that. However, I haven't read through the mission file yet, so if there's something there that justifies this response (becoming respected with the Unfettered by the time this job becomes available to you, some aspect of Unfettered culture I haven't seen) then please let me know.

Reply:
Well he is very angry to have lost but you did just break his spirit by disabling his ship, this is a duel after all, the intention was to fight you, and maybe lose. I really do not want the player to get the impression he could kill or capture this ship without consequences, also the "duellists" do not use the same hails as the rest of the Unfettered when they are hostile (given the idea is they ask you to fight them), they are only annoyed, not provocative.

Reply:
That said, this one is the least experienced of all of the captains you fight, so I do agree he could be a bit more aggressive.
I tried to have them react different ways (not happy to "wise")

Reply:
I changed the first 2 dialogs from the captains, the first one is clearly angry now, second one is neutral and third one stays positive